commit 9af18a94e051fce513972254bf97c12ae563841e
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Sun Feb 22 23:34:25 2026 +0100

    hurd: Define _POSIX_TIMERS to 200809L
    
    We now have monotonic vs realtime clocks, and high-precision.

diff --git a/sysdeps/mach/hurd/bits/posix_opt.h b/sysdeps/mach/hurd/bits/posix_opt.h
index a1a0707d25..6a6b346a31 100644
--- a/sysdeps/mach/hurd/bits/posix_opt.h
+++ b/sysdeps/mach/hurd/bits/posix_opt.h
@@ -150,8 +150,8 @@
 /* The `spawn' function family is supported.  */
 #define _POSIX_SPAWN	200809L
 
-/* We do not have POSIX timers, but could in future without ABI change.  */
-#define _POSIX_TIMERS	0
+/* We have POSIX timers.  */
+#define _POSIX_TIMERS	200809L
 
 /* The barrier functions are available.  */
 #define _POSIX_BARRIERS	200809L
